imports/plugins/core/orders/client/components/orderTable.js

```diff
@@ -108,53 +111,49 @@ class OrderTable extends Component {
     });
 
     return (
-      <SortableTable
-        tableClassName="rui order table -highlight"
-        publication="CustomPaginatedOrders"
-        collection={Orders}
-        matchingResultsCount="order-count"
-        columnMetadata={customColumnMetadata}
-        externalLoadingComponent={Loading}
-        filterType="none"
-        getTheadProps={() => {
-          return {
-            style: {
-              borderTop: "1px solid #e6e6e6",
-              borderRight: "1px solid #e6e6e6",
-              borderLeft: "1px solid #e6e6e6"
-            }
-          };
-        }}
-        getTrGroupProps={() => {
-          return {
-            style: {
-              borderRight: "1px solid #e6e6e6",
-              borderLeft: "1px solid #e6e6e6"
-            }
-          };
-        }}
-        getTrProps={() => {
-          return {
-            style: {
-              height: 49,
-              maxHeight: 49
-            }
-          };
-        }}
-        getPaginationProps={() => {
-          return {
-            className: "order-table-pagination"
-          };
-        }}
-        getTableProps={() => {
-          return {
-            style: {
-              borderBottom: "1px solid #e6e6e6"
-            }
-          };
-        }}
-        showPaginationTop={true}
-      />
+      <div>
+        {renderBulkOrderActionsBar()}
+        <SortableTable
+          tableClassName={`rui order table ${renderTableClassNameHidden()} -highlight`}
+          publication="CustomPaginatedOrders"
+          collection={Orders}
+          matchingResultsCount="order-count"
+          columnMetadata={customColumnMetadata}
+          externalLoadingComponent={Loading}
+          filterType="none"
+          selectedRows={selectedItems}
+          getTheadProps={() => {
+            return {
+              style: {
+                borderTop: "1px solid #e6e6e6",
+                borderRight: "1px solid #e6e6e6",
+                borderLeft: "1px solid #e6e6e6"
+              }
+            };
+          }}
+          getTrGroupProps={() => {
+            return {
+              style: {
+                borderRight: "1px solid #e6e6e6",
```

Comment:
Is there any way to not hard-code these values in? (I know you are sort of "inheriting" this code)